### ISSUE TITLE
ReferenceError: ad_id is not defined

### ISSUE THREAD

**Issue by Stetocasa** [OpenWonderLabs/homebridge-switchbot-ble#249]
**Describe The Bug:**

After last update no bot discovered from the plug-in 
**To Reproduce:**

I’m looking for a support to solve the issue 

no

```

27/8/2021, 21:40:14Switchbot Lock BridgeSwitchBotPlatformCurrent state of Bot was returned: OFF
27/8/2021, 21:40:18Switchbot Lock BridgeSwitchBotPlatformTarget state of Bot setting: ON
27/8/2021, 21:40:19Switchbot Lock BridgeERRORprocessLeAdvertisingReport: Caught illegal packet (buffer overflow): ReferenceError: ad_id is not defined
27/8/2021, 21:40:20Switchbot Lock BridgeSwitchBotPlatformScan done.
27/8/2021, 21:40:20Switchbot Lock BridgeSwitchBotPlatformNo device was found during scan.
27/8/2021, 21:40:20Switchbot Lock BridgeSwitchBotPlatformERRORError: No device was found during scan.
27/8/2021, 21:40:20Switchbot Lock BridgeSwitchBotPlatformBot state failed to be set to: ON

***Read and Paste over this Text the Text Above^^^^^^
```

**Homebridge Config:**

```

}

    "platform": "SwitchBotPlatform",
    "name": "SwitchBotPlatform",
    "devices": [
        {
            "type": "bot",
            "name": "Portone",
            "bleMac": "c0:04:36:23:E5:77",
            "scanDuration": 2000
        },
        {
            "type": "bot",
            "name": "Beko Pause",
            "bleMac": "df:0A:A6:E9:AE:90",
            "scanDuration": 2000
        }
    ]
}

```

**Screenshots:**

**Device Model:**
Switchbot

# **Environment:**

- **Node.js Version**: 14.17.5
- **NPM Version**: 
- **Homebridge Version**: 1.3.4 + Hoobs 4.0.100
- **Homebridge SwitchBot BLE Plugin Version**: 1.6.0

**Comment by tobiwashere** [comment]
Same here with version 1.6.0.
Version 1.5.1 doesn't contain these errors.

### My environment:
**Node.js Version:** 14.17.4
**NPM Version:** 6.14.14
**Homebridge Version:** 1.3.4 standalone
**Homebridge SwitchBot BLE Plugin Version:** 1.6.0
**Device Model:** 2x Switchbot Curtain

**Comment by donavanbecker** [comment]
fixed with v1.6.1
